--- origsrc/font-manager-0.5.3/src/constants.py.in.in	2010-05-13 13:25:28.000000000 -0500
+++ src/font-manager-0.5.3/src/constants.py.in.in	2010-05-30 00:29:41.055382900 -0500
@@ -54,7 +54,7 @@ DESKTOP_DIR                 =   glib.get
                                 (glib.USER_DIRECTORY_DESKTOP)
 
 HOME                        =   os.getenv('HOME')
-USER                        =   os.getenv('LOGNAME')
+USER                        =   os.getenv('USER')
 USER_FONT_DIR               =   join(HOME, '.fonts')
 USER_FONT_COLLECTIONS       =   join(DATA_DIR, 'Collections.xml')
 USER_FONT_COLLECTIONS_BAK   =   join(DATA_DIR, 'Collections.xml.bak')
--- origsrc/font-manager-0.5.3/src/ui/sampler.py	2010-05-13 13:25:27.000000000 -0500
+++ src/font-manager-0.5.3/src/ui/sampler.py	2010-05-30 00:49:07.039073400 -0500
@@ -69,7 +69,7 @@ class Config(object):
     def __init__(self):
         self.styles = getSampleStyleSheet()
         self.fontsize = 20
-        author =  os.getenv('LOGNAME')
+        author =  os.getenv('USER')
         self.author = author.capitalize()
         self.subject = _('Sample of included fonts')
         self.pangram = False
